src/tests/test_api/test_views_verblijfplaatshistorie.py

```diff
@@ -86,3 +88,42 @@ def test_bsn_date_search_deny(self, api_client, common_headers):
             "title": "You do not have permission to perform this action.",
             "type": "https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.3",
         }
+
+    def test_encrypted_bsn(self, api_client, requests_mock, common_headers, caplog):
+        """Prove that search is possible"""
+        requests_mock.post(
+            # https://demo-omgeving.haalcentraal.nl
+            "/haalcentraal/api/brphistorie/verblijfplaatshistorie",
+            json=self.RESPONSE_VERBLIJFPLAATS,
+            headers={"content-type": "application/json"},
+        )
+
+        url = reverse("brp-verblijfplaatshistorie")
+        token = build_jwt_token(["benk-brp-verblijfplaatshistorie-api", "benk-brp-encrypt-bsn"])
+
+        # Create an encrypted BSN to use in the request
+        encrypted_bsn = encryption.encrypt("999993240")
```

Comment:
wellicht een assert wat ongeveer de waarde is? (bijvoorbeeld een `startswith()`)

Reply:
Nog even toegevoegd, de waarde lijkt nu in ieder geval altijd hiermee te beginnen.